apps/etterlatte-brev-api/src/main/kotlin/no/nav/etterlatte/brev/model/EndringBrevData.kt

```diff
@@ -13,8 +13,6 @@ import java.time.LocalDate
 abstract class EndringBrevData : BrevData()
 
 data class EtterbetalingDTO(
-    val fraDato: LocalDate,
```

Comment:
OMS bruker fra og tilDato i sitt vedlegg om etterbetaling så er det noen tanker om å legge disse feltene tilbake senere eller vil det løses på en annen måte?

Reply:
> OMS bruker fra og tilDato i sitt vedlegg om etterbetaling så er det noen tanker om å legge disse feltene tilbake senere eller vil det løses på en annen måte?

Det ser ut for meg som dokker gjer det i `forEach(beregningsperioder) {`, og ikkje på `EtterbetalingDTO`-objektet som sådan, så det bør vera upåverka. Med mindre eg har oversett noko?

Reply:
Første setning i vedlegget inneholder fraDato og tilDato 
```
paragraph {
      textExpr(
          Bokmal to "Du får etterbetalt stønad fra ".expr() + fraDato.format() + " til " + tilDato.format()  +
                  ". Vanligvis vil du få denne etterbetalingen i løpet av tre uker. ",
          Nynorsk to "".expr(),
          English to "".expr(),
      )
}
```

Men man vet aldri om det vil endre seg før det går i prod 🤷

Reply:
Aah. Da legg eg det tilbake igjen.

Reply:
Ah. Det fekk gå uoppdaga fordi vi i pensjonsbrev har to klassar EtterbetalingDTO. Da heng ting meir på greip.

Samlar dei to til ein med det samme, ser ingen grunn til at den skal divergere.